checks/evaluation/pinned_dependencies.go

```diff
@@ -121,14 +121,21 @@ func PinningDependencies(name string, c *checker.CheckRequest,
 		return checker.CreateRuntimeErrorResult(name, err)
 	}
 
+	// Pip installs.
+	pipScore, err := createReturnForIsPipInstallPinned(pr, dl)
```

Comment:
what happens to the other Npm, curl, etc? Are they also not accounted for in the final score?

Reply:
Yes, they are also not accounted in the final score. I plan on including npm, choco and go commands in separate PRs to count for the final score. curl is a different case, it already counts in the final score.

Reply:
SG, thanks @gabibguti